internal/types/keysign.go

```diff
@@ -27,12 +29,18 @@ func (r KeysignRequest) IsValid() error {
 	if len(r.Messages) == 0 {
 		return errors.New("invalid messages")
 	}
+	if r.Key == "" {
```

Comment:
For keysign , you don't need this `Key` , the local party id is part of the Vault